content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/count_products.rst

```diff
@@ -106,35 +123,47 @@ that the quantity on hand before the adjustment has not yet been updated to matc
 counted quantity.
 
 There are two ways to apply the new inventory adjustment. The first way is to click the
-:guilabel:`Apply` button on the line at the far right of the page. The second way is to click the
-checkbox on the far left of the line. Doing so reveals new button options at the top of the page,
-one of which is an :guilabel:`Apply` button. Clicking this button instead causes an
+:icon:`fa-save` :guilabel:`Apply` button on the line at the far right of the page. The second way is
+to click the checkbox on the far left of the line. Doing so reveals new button options at the top of
+the page, one of which is an :guilabel:`Apply` button. Clicking this button instead causes an
 :guilabel:`Inventory Adjustment Reference / Reason` pop-up window to appear.
 
+.. image:: count_products/apply-inventory-adjustment.png
+   :align: center
+   :alt: Apply all option applies the inventory adjustment once a reason is specified.
+
 From this pop-up menu, a reference or reason can be assigned to the inventory adjustment. By
 default, the :guilabel:`Inventory Reference / Reason` field is pre-populated with the date the
 adjustment is being made on, but can be changed to reflect whatever reference or reason is desired.
 
 Once ready, click :guilabel:`Apply` to apply the inventory adjustment.
 
 .. note::
-   Applying an inventory adjustment simultaneously creates a :doc:`stock move line (SML)
-   <../reporting/moves_history>` in the *Moves History* report for traceability.
+   Applying an inventory adjustment simultaneously creates a
+   :doc:`stock move line (SML) <../reporting/moves_history>` in the *Moves History* report for
+   traceability.
 
-.. image:: count_products/apply-inventory-adjustment.png
-   :align: center
-   :alt: Apply all option applies the inventory adjustment once a reason is specified.
+.. tip::
+   Sometimes products end up in unexpected or incorrect locations. To quickly move products to a
+   different storage location, click the :guilabel:`Relocate` button at the top of the
+   :ref:`Inventory Adjustments <inventory_management/count_products/inventory-adjustments-page>`
+   page.
 
 Count products
 ==============
 
 Counting products is a recurring activity in a warehouse. Once a count is complete, go to
-:menuselection:`Inventory app --> Operations --> Inventory Adjustments` to update the
+:menuselection:`Inventory app --> Operations --> Physical Inventory` to update the
 :guilabel:`Counted Quantity` column for each product line.
 
+To print a PDF of a count sheet, add products to the count sheet by ticking their checkboxes. Click
+the :icon:`fa-print` :guilabel:`(printer icon)`, then select :guilabel:`Count sheet` to
```

Comment:
When an icon accompanies text, we don't need to describe the icon in guilabels (: 
```suggestion
the :icon:`fa-print` :guilabel:`Print` button at the top, then select :guilabel:`Count sheet` to
```

Reply:
Hmm, I think I see what you're saying. I was trying to follow this first descriptive icon style from the RST guidelines: 
![image](https://github.com/user-attachments/assets/a4988783-1487-41f8-b37b-896c161237b4)

Reply:
Ahhh, I went back to the UI, and I see what you mean! I hadn't realized it said 'Print' next to the icon. Thanks for pointing that out! :)